src/homework19/GaragePage/GaragePage.js

```diff
@@ -0,0 +1,20 @@
+import BasePage from "../BasePage.js";
+import {SideBarMenu} from "./components/SideBarMenu.js";
+import Header from "../../components/Header.js";
+
+
+export default class GaragePage extends BasePage{
+    constructor(page) {
+        super(page, "/panel/garage");
+        this.titleGaragePage =  page.locator('h1', {hasText: 'Garage'});
```

Comment:
do it this way
`this.sideBar = new SideBarMenu(this._page)`

Reply:
done)